primer.md

```diff
@@ -187,6 +188,13 @@ not specific to CloudEvents and can be used to post any kind of one-way event
 and notifications to a conformant HTTP endpoint. However, the lack of such a
 specification elsewhere makes it necessary for CloudEvents to define it.
 
+### Transport Error Handling
+
+The CloudEvents specification, for the most part, does not dictate a processing
+model associated with the creation or processing of CloudEvents. As such, if
+there are errors during the processing of a CloudEvent it is assumed that the
+normal transport-level error reporting mechanisms will be used.
```

Comment:
I don't think this is good enough advice. Should the producer retry? Should it propagate the error downstream? how does it propagate errors that cross transports??

Reply:
IMO those are semantic decisions that are out of scope of the spec. To me, CE is about adding metadata to an event to help with routing/filtering/etc.  In the most basic form, binary, we add a few bits of metadata (e.g. 4 http headers) and that's it. How retries or errors are handled do not change just because we said "add 4 new ce-xxx headers".  The questions you're asking were there before CE and they'll be there after CE because we aren't trying to solve those problems.  

Our git repo README says:
> CloudEvents is a specification for describing event data in common formats to provide interoperability across services, platforms and systems.

"describing event data" - not "transporting events". Or put another way, we're not defining an eventing/messaging transport framework spec. Follow-on specs/projects could choose to work on those though.  I'm not dismissing there might be issues with what you're concerned about, I'm just saying it's out of scope for what we agreed to focus on.

Reply:
And this PR tries to make it clear that yes we are "punting"  :-)

Reply:
Could we say something to be a bit more proactive about encouraging people to report problems?  In typical eventing infrastructure there won't be a direct channel to feedback to a sender of broken messages, so good citizenship of the ecosystem requires making noise when receiving broken data. So I'd change the language to say something like "if there are errors during the processing of a CloudEvent, the software encountering the error is encouraged to use the normal transport-level error reporting to report them."